apiary.apib

```diff
@@ -2881,6 +2881,70 @@ Update a milestone.
 
 + Response 204 (application/json;charset=utf-8)
 
+## Project items [/projectItems]
+
+Project items are added to a project and they affect the profitability. This information is only available for account administrators and project decision makers.
+
+*Required scopes: `projects`*
+
+### projectItems.list [GET /projectItems.list]
+
+Get a list of project items.
+
++ Request (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + filter (object, optional)
+            + project_id: `082e6289-30c5-45ad-bcd0-190b02d21e81` (string, optional)
+        + page (Page, optional)
+        + sort (array, optional)
+            + (object)
+                + field (enum[string], required)
+                    + Members
+                        + date
+                + order (enum[string], optional)
+                    + Members
+                        + asc
+                        + desc
+            + Default
+                + (object)
+                    + field: `date`
+                    + order: `desc`
+
+
++ Response 200 (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + data (array)
+            + (object)
+                + id: `cfb4146d-06be-41f1-bb39-aa3c929c71dc` (string)
+                + project (object)
+                    + type: `project` (string)
+                    + id: `944534fb-15f1-4eea-aab1-82a427aa2d0d` (string)
+                + milestone (object, nullable)
+                    + type: `milestone` (string)
+                    + id: `944534fb-15f1-4eea-aab1-82a427aa2d0d` (string)
+                + type: `time_tracked` (enum[string])
+                    + Members
+                        + time_tracked
+                        + external_cost_item
+                + description: `Drafted the document` (string, nullable)
+                + date: `2018-11-26T16:44:33+00:00` (string)
```

Comment:
What does this date represent?

Reply:
The date most relevant for the project item. In case of time tracked: the date on which it was performed. For an order item: the date it was added.

Reply:
Can we find a better name than just `date` for his concept?

Reply:
Any proposals? :D I have 'item_date' / 'happened_at' / 'date_accomplished' / 'accomplished_at' / 'relevant_date' but I still feel just 'date' is better :)

Reply:
Unless there's any other proposal, I will rename it to 'item_date'.

@jenssegers @adognameddog @mrtus @rathesDot @tiagobastos

Reply:
I don't think `item_date` explains so much what it is about, the only thing I can come up with is `date_of_occurence` but I don't think that is any better

Reply:
We decided to call it `occurred_at`